src/components/MetricsPane/charts/AggregateHardCoralCover.jsx

```diff
@@ -12,28 +12,42 @@ export const AggregateHardCoralCover = () => {
   const { filteredSurveys } = useContext(FilterProjectsContext)
 
   const hardCoralAveragesPerSurvey = filteredSurveys
-    .map((record) => {
-      const { protocols } = record
-
-      const benthicPitAverage =
-        protocols.benthicpit?.percent_cover_benthic_category_avg?.['Hard coral'] ?? 0
-      const benthicLitAverage =
-        protocols.benthiclit?.percent_cover_benthic_category_avg?.['Hard coral'] ?? 0
-      const benthicPqtAverage =
-        protocols.benthicpqt?.percent_cover_benthic_category_avg?.['Hard coral'] ?? 0
-      const quadratBenthicAverage = protocols.quadrat_benthic_percent?.percent_hard_avg_avg ?? 0
-
-      const numerator =
-        benthicLitAverage + benthicPitAverage + benthicPqtAverage + quadratBenthicAverage
-      const denominator =
-        (benthicPitAverage ? 1 : 0) +
-        (benthicLitAverage ? 1 : 0) +
-        (benthicPqtAverage ? 1 : 0) +
-        (quadratBenthicAverage ? 1 : 0)
-
-      return !denominator ? undefined : numerator / denominator
+    .map(({ protocols }) => {
+      const getHardCoralCover = (data, protocol) =>
+        data?.[protocol]?.percent_cover_benthic_category_avg?.['Hard coral']
+
+      const benthicPitCover = getHardCoralCover(protocols, 'benthicpit')
+      const benthicLitCover = getHardCoralCover(protocols, 'benthiclit')
+      const benthicPqtCover = getHardCoralCover(protocols, 'benthicpqt')
+      const quadratBenthicCover = protocols?.quadrat_benthic_percent?.percent_hard_avg_avg
+
+      if (
+        benthicPitCover === undefined &&
+        benthicLitCover === undefined &&
+        benthicPqtCover === undefined &&
+        quadratBenthicCover === undefined
```

Comment:
Why are we being so specific about undefined and not capturing null or empty strings, etc?